Source/Core/Common/StringUtil.cpp

```diff
@@ -524,8 +525,11 @@ std::string CodeTo(const char* tocode, const char* fromcode, std::basic_string_v
     while (src_bytes != 0)
     {
       size_t const iconv_result =
+#if defined(__OpenBSD__) || defined(__NetBSD__)
+          iconv(conv_desc, (const char**)&src_buffer, &src_bytes, &dst_buffer, &dst_bytes);
```

Comment:
```suggestion
          iconv(conv_desc, reinterpret_cast<const char**>(&src_buffer), &src_bytes, &dst_buffer, &dst_bytes);
```
(you might need to re-run clang-format after applying this suggestion)